mne/forward/_field_interpolation.py

```diff
@@ -87,6 +105,90 @@ def _compute_mapping_matrix(fmd, info):
     return mapping_mat
 
 
+def _as_meg_type_evoked(evoked, to_type='grad', mode='fast', n_jobs=1):
```

Comment:
I think eventually we'll want to separate the evoked-ness of this from the remapping portion. Basically all you really need to do the remapping is an info struct and data. And even the data could be omitted and this function could return the remapping matrix to `dot` with potentially. It would make it so that we could more easily map data of any type (raw, epochs, evoked) if we want. But that can be saved for a later PR.